alot/commands/globals.py

```diff
@@ -724,7 +724,7 @@ class ComposeCommand(Command):
     def __init__(
             self,
             envelope=None, headers=None, template=None, sender=u'',
-            tags=None, subject=u'', to=None, cc=None, bcc=None, attach=None,
+            tags=None, subject=None, to=None, cc=None, bcc=None, attach=None,
```

Comment:
when using `alot compose`, the default is overriden with None which caused me some trouble so better default to None IMO.